go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/jaekwon/openpgp
+
+go 1.23.8
```

Comment:
There are fixes and improvements to the `crypto` packages in later Go versions (1.24 and 1.25), some of them protect against some mathematical attacks on the `crypto/rsa` package for example. See: https://go.dev/doc/go1.25#cryptorsapkgcryptorsa

Should we consider upgrading?